docs/connectors/slack.md

```diff
@@ -74,15 +76,12 @@ You need to choose between two backends. The [Events API](https://api.slack.com/
 
 If you are unsure which one is the best for you, [Slack Faq](https://api.slack.com/faq#events_api) provide differences between those two.
 
-```eval_rst
-.. note::
-  You should follow the instructions for the Event API first when configuring your slack connector, even if you are planning
-  on using slack in Socket Mode.
+```{note}
+While it is not mandatory to configure the Event API before using Slack in Socket Mode, we recommend
+setting up the Event API for its full feature set.
```

Comment:
This is MyST syntax, but unfortunately we do not use that here. Please can you revert this exactly as it was.

````markdown
```eval_rst
.. note::
   ...
```
````

Reply:
This is how the note was formatted before my changes:
```{note}
```{note}
You should follow the instructions for the Event API first when configuring your slack connector, even if you are planning
on using slack in Socket Mode.
```

Shoudn't I put it like this again?